src/ansys/fluent/core/services/svar.py

```diff
@@ -435,11 +435,12 @@ def __init__(
         self._service = service
         self._svar_info = svar_info
 
```

Comment:
In the old version, we do the work once in the constructor. In the new version, we do the work on demand and we want to ensure that we still only do it once. But, there seems to be no check about that.

Reply:
Right @seanpearsonuk. However, we don't want to do it once as it fetches the Svars Info every time there are some updates to the data. That's why the check has been removed now. Are we missing something?

cc. @ajain-work

Reply:
OK, I was just comparing this with the original code, which I suppose was wrong in this respect.

Reply:
Yes, @seanpearsonuk. Thanks to @ajain-work for pointing out this issue :).

Reply:
OK, thanks for clarifying. It completely makes sense - it was only that it was a bit mysterious. Anyway, no problem, let's proceed.